recipes/pynput/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pynput" %}
+{% set version = "1.6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b8a6efac6e50ca86d0354da2070a07826eec46a1e0c02f47ef09523d380a734d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
```

Comment:
If with adding `python` in the `run` section you are still getting the same error, try adding `setuptools` and `sphinx >=1.3.1` in `host` too.